src/_nebari/render.py

```diff
@@ -153,7 +153,7 @@ def list_files(
             deleted_files.add(path)
 
     for filename in list_files(output_base_dir, ignore_filenames, ignore_directories):
-        relative_path = pathlib.Path(os.path.relpath(filename, output_base_dir))
+        relative_path = str(pathlib.Path(os.path.relpath(filename, output_base_dir)))
```

Comment:
Fix for a bug related to mixed use of PosixPath and str values in the lists being returned

Reply:
I resolved this issue upstream using `pathlib.Path.relative_to` so we and remove this.